src/server/controllers/s3.js

```diff
@@ -0,0 +1,44 @@
+const router = require('express').Router
+const ms = require('ms')
+const S3 = require('aws-sdk/clients/s3')
+
+const defaultConfig = {
+  acl: 'public-read',
+  key: process.env.UPPYSERVER_AWS_KEY,
```

Comment:
maybe we should pass the default env variables via the [stand-alone server](https://github.com/transloadit/uppy-server/blob/master/src/standalone.js#L56) and not have the pluggable app worry about environment variables?